### PR TITLE
Update A Quick Start Guide to Building ZAP

### PR DESCRIPTION
**Pull request by Avinash-Acharya**
Making modifications to the **"A QUICK START GUIDE TO BUILDING ZAP"** page. 
Changes include adding a depth to the PR segment of the document by adding a few sentences and redirecting to a *"Guideline on PR Submission"*.

Signed-off-by: Avinash Acharya <120123021+Avinash-Acharya@users.noreply.github.com>

### CODE REVIEW

File: site/content/docs/developer/quick-start-build.md [hunk 1]
```diff
@@ -170,4 +170,6 @@ cd ..
 ```
 
 ### Make Changes and Submit a PR!
+We welcome any changes made by a Contributor following a review process conducted by a ZAP core team member. Prior to submittion of a PR, A contributor must strictly adhere to the [Guidelines](https://github.com/zaproxy/zaproxy/blob/c6fbfcb2035f5a452af59a1a9080855c06bde4c0/CONTRIBUTING.md#guidelines-for-pull-request-pr-submission-and-processing).
```
Comment (kingthorin) [inline site/content/docs/developer/quick-start-build.md:173]:
```suggestion

We welcome any changes made by a Contributor following a review process conducted by a ZAP core team member. Prior to submission of a PR, please be aware contributions must adhere to the [Guidelines](https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#guidelines-for-pull-request-pr-submission-and-processing).
```

**Review by ricekot — COMMENTED** [review]
Other than the line on "follow the guidelines", this PR is simply paraphrasing the existing text and that is really a matter of preference.

**Comment by Avinash-Acharya** [comment]
> Other than the line on "follow the guidelines", this PR is simply paraphrasing the existing text and that is really a matter of preference.

Ya exactly , for some reason I found it necessary at that moment . I will make sure not to make unnecessary chances.

File: site/content/docs/developer/quick-start-build.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ date: "2020-12-04"
 addSocialPreview: true
 ---
 
-Have you ever wanted to play around with the ZAP codebase but felt it was too overwhelming? This guide will walk you through building ZAP from the command line regardless of the operating system and IDE you're using.
+Ever wanted to experiment with the ZAP codebase but felt intimidated by it? This guide will walk you through the process of building ZAP from the command line regardless of your operating system or IDE of your choice.
```
[the remark this answers is not in the document]
Comment (Avinash-Acharya) [reply site/content/docs/developer/quick-start-build.md:14]: Should I bring the original statement back or just remove the "of your choice" ?

**Comment by Avinash-Acharya** [comment]
I made sure to modify according to the suggestion.

**Review by kingthorin — COMMENTED** [review]
LGTM

**Review by kingthorin — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

File: site/content/docs/developer/quick-start-build.md [hunk 3]
```diff
@@ -170,4 +170,7 @@ cd ..
 ```
 
 ### Make Changes and Submit a PR!
+
+We welcome any changes made by a Contributor following a review process conducted by a ZAP core team member. Prior to submission of a PR, please be aware: Contributions must adhere to the [Guidelines](https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#guidelines-for-pull-request-pr-submission-and-processing).
```
Comment (thc202) [inline site/content/docs/developer/quick-start-build.md:174]: I'd prefer if this didn't go into details saying how many core team members do the reviews, as is now it seems that a single reviewer is enough but as mentioned in the linked page it's usually required two.

Comment (kingthorin) [reply site/content/docs/developer/quick-start-build.md:174]: > We welcome any changes made by a Contributor following a review process. Prior to submission of a PR, please be aware: Contributions must adhere to the [Guidelines](https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#guidelines-for-pull-request-pr-submission-and-processing).

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by Avinash-Acharya** [comment]
Thanks...!!

**Pull request merged**